autoscheduler/frontend/src/timeUtil.ts

```diff
@@ -0,0 +1,14 @@
+export const FIRST_HOUR = 8;
+export const LAST_HOUR = 21;
+/**
+ * Given the hours and minutes of a time, returns a formatted string representation of
+ * that time. By default, this uses 24-hour format, but an extra argument can be specified
+ * to use 12-hour format
+ * @param h hours
+ * @param m minutes
+ * @param use12Hour if true, applies 12-hour format instead of the default 24-hour format
+ */
+export function formatTime(h: number, m: number, use12Hour = false): string {
```

Comment:
I actually had one in there, but ESLint automatically removes it if you provide a default value, since the type can be inferred from that of the default value